tests/acceptance/features/apiSpaces/search.feature

```diff
@@ -109,3 +109,15 @@ Feature: Search
     Then the HTTP status code should be "207"
     And the search result should contain "1" entries
     And for user "Alice" the search result should contain space "find data"
+
+
+  Scenario: user can search inside folder in space
+    When user "Alice" searches for "folder" inside folder "/folderMain" in space "find data" using the WebDAV API
```

Comment:
space name was there already but we should prevent using names like `find data`
```suggestion
    When user "Alice" searches for "folder" inside folder "/folderMain" in space "<some-good-name>" using the WebDAV API
```

Reply:
so should i make a new space in this scenario only or make change in the background? But if we make change in the background it should be corrected on all the scenario where it is used.(probably make a new pr for this)

Reply:
fine for now. we can do that renaming in next PRs